cargo-miri/src/util.rs

```diff
@@ -88,6 +88,77 @@ pub fn escape_for_toml(s: &str) -> String {
     format!("\"{s}\"")
 }
 
+pub fn flagsplit(flags: &str) -> Vec<String> {
+    // This code is taken from `RUSTFLAGS` handling in cargo.
+    // Taken from miri-script util.rs
+    flags.split(' ').map(str::trim).filter(|s| !s.is_empty()).map(str::to_string).collect()
+}
+
+pub fn get_miriflags_cargo_mini() -> Vec<String> {
+    // Respect `MIRIFLAGS` and `miri.flags` setting in cargo config.
+    // If MIRIFLAGS is present, flags from cargo config are ignored.
+    // This matches cargo behavior for RUSTFLAGS.
```

Comment:
Please add a link to where this behavior is documented for cargo.

Reply:
Hi Ralf, this is from PR #2451 https://github.com/rust-lang/miri/pull/2451/files#diff-922365a8fd004632c70c2426b2851c89ed2e13229b7e7693550b87f00b100badR552 for the function get_miriflags().

I'll remove this block of comment from this function.

The comment can be made clearer by specifying `flags from cargo config` to `flags from .cargo/config.toml` (configuration file instead of --config option) since we run this in `get_miriflags_runner`
```rust
    let mut cmd = cargo();
    cmd.args(["-Zunstable-options", "config", "get", "miri.flags", "--format=json-value"]);
```

The behavior is from https://doc.rust-lang.org/cargo/reference/config.html#command-line-overrides

> Cargo also accepts arbitrary configuration overrides through the --config command-line option. The argument should be in TOML syntax of KEY=VALUE:
`cargo --config net.git-fetch-with-cli=true fetch`
The --config option may be specified multiple times, in which case the values are merged in left-to-right order, using the same merging logic that is used when multiple configuration files apply. 
Configuration values specified this way take precedence over environment variables, which take precedence over configuration files.

We check the arguments from --config first, then environment variable, then check the configuration files via `cargo config get`.

I couldn't find the evidence for `This matches cargo behavior for RUSTFLAGS.` Should I remove that line?